otm-gateway.html

```diff
@@ -248,12 +286,19 @@ <h3><dfn>Retrieve Object</dfn></h3>
           Date: Tue, 02 Jul 2019 20:45:00 GMT
           Content-Length: 0
         </pre>
+        <pre class="example" title="Retrieve Object Version Request">
+          GET /af48c3d?versionId=20190702T201500.001 HTTP/1.1
+          Host: preservation-gateway.institution.edu
+          Date: Tue, 02 Jul 2019 20:45:00 GMT
+          Content-Length: 0
+        </pre>
         <pre class="example" title="Retrieve Object Response">
           HTTP/1.1 200 OK
           Date: Tue, 02 Jul 2019 20:45:00 GMT
           ETag: "4efcb3d98ce0fabfd585eb6c4332859"
           Content-Length: 493285
```

Comment:
Just curious here: On a restore the Bridge provides an API call for the Gateway to GET each individual file that is included in the restore request. Here in the Gateway, the only call is to GET the entire object. I'm assuming, then, that the Gateway will pull all the files from the Bridge and package them together in a way that allows them all to be handed back to the repository as a single bitstream. The repository would then know how to extract the package and put the individual files where they should go. Does this sound like the flow you're expecting?

Reply:
that's right.